scripts/src/main/resources/scripts/command/jenkins

```diff
@@ -12,48 +12,67 @@ fi
 
 # shellcheck source=scripts/functions
 source "$(dirname "${0}")"/../functions
-JENKINS_HOME="${DEVON_IDE_HOME}/software/jenkins"
-TOOL_VERSION_COMMAND="java -jar '${JENKINS_HOME}/jenkins.war' --version"
+JENKINS_CLI_HOME="${DEVON_IDE_HOME}/software/jenkins"
```

Comment:
mhm, this `«TOOL»_HOME` environment variables seems to cause some confusion.
Typically this refers to the installation folder of that tool.
This applies for `JAVA_HOME`, `MAVEN_HOME`, `GRADLE_HOME`, etc.
For CobiGen we seemed to have the first excuse where somebody (I guess you @jan-vcapgemini) explained that CobiGen treats `COBIGEN_HOME` as the directory where the user specific configuration for CobiGen is stored that is usually located somewhere in the users home directory.
I still think that such things are odd and it would be better if all tools would follow a common scheme how to name environment variables (and I am not saying that `«TOOL»_HOME` was really a great choice but it became more or less a de-facto standard and not following such de-facto standard is typically not great).
However, I did some research and found [this one link](https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/Jenkins-Home-Directory-Location-Change-JENKINS_HOME-windows-linux-ubuntu) that says you are absolutely correct that `JENKINS_HOME` means the `.jenkins` folder that is by default in the users home directory. On the other hand `JENKINS_CLI_HOME` indicates to be the directory where the `jenkins-cli` is "at home". However, the main thing here is the `jenkins.war` file and if something represents jenkins then it is this war file.
Anyhow, I have no better clue so we should just follow your suggestion and rename it this way.
(I guess when we go to Java such internal variables (e.g. `JENKINS_CLI_HOME` will disappear anyhow.)